docs/alternatives/index.md

```diff
@@ -225,11 +259,19 @@ At this point, you've assessed:
 Now we're ready to compare the libraries "as units libraries" --- that is, in terms of their core
 features.
 
+!!! note
+    The features are listed, _very_ roughly, in order of importance.  Counting up the colours in
```

Comment:
Do we use British spelling?

Reply:
Chuffed though I'd be if we did, we don't. "Colors" it is.

Reply:
Canadian habits!  :grin: 

This probably isn't the last to be found among our docs.